launcher/src/CmderLauncher.cpp

```diff
@@ -113,8 +113,21 @@ void StartCmder(std::wstring path, bool is_single_mode)
 	PathRemoveFileSpec(exeDir);
 
 	PathCombine(icoPath, exeDir, L"icons\\cmder.ico");
-	PathCombine(oldCfgPath, exeDir, L"config\\ConEmu.xml");
-	PathCombine(cfgPath, exeDir, L"vendor\\conemu-maximus5\\ConEmu.xml");
+
+	// Check for machine-specific config file.
+	PathCombine(oldCfgPath, exeDir, L"config\\ConEmu-%COMPUTERNAME%.xml");
```

Comment:
Should this be "config\user-ConEmu-%COMPUTERNAME%.xml" so it will be ignored by '.gitignore'?